cinema/serializers.py

```diff
@@ -1 +1,108 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Movie,
+    Genre,
+    Actor,
+    CinemaHall,
+    MovieSession,
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = [
+            "id",
+            "first_name",
+            "last_name",
+            "full_name"
```

Comment:
The `full_name` field in `ActorSerializer` is a property and not a model field, which means it cannot be serialized directly using `ModelSerializer`. You should define it as a `SerializerMethodField` and implement a method to return the full name.